src/sonic-config-engine/minigraph.py

```diff
@@ -484,6 +488,39 @@ def parse_xml(filename, platform=None, port_config_file=None):
 
     return results
 
+def parse_device_desc_xml(filename):
+    root = ET.parse(filename).getroot()
+    (lo_prefix, mgmt_prefix, hostname, hwsku, d_type) = parse_device(root)
+
+    Tree = lambda: defaultdict(Tree)
+    results = Tree()
```

Comment:
Fixed.

---
In reply to: [125150653](https://github.com/Azure/sonic-buildimage/pull/775#discussion_r125150653) [](ancestors = 125150653)